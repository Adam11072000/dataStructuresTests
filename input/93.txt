init 193
resetLabelScore 129 -9 135
resetLabelScore -3 102 192
getHighestScoredLabel 244 -8
mergeSuperPixels 181 188 -6
resetLabelScore 94 30 -6
resetLabelScore 30 21 -10
resetLabelScore 239 73 52
quit