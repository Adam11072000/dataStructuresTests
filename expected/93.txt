init done.
resetLabelScore: INVALID_INPUT
resetLabelScore: INVALID_INPUT
getHighestScoredLabel: INVALID_INPUT
mergeSuperPixels: INVALID_INPUT
resetLabelScore: INVALID_INPUT
resetLabelScore: INVALID_INPUT
resetLabelScore: FAILURE
quit done.
